### PR TITLE
Update readme to v0.15

### PR DESCRIPTION
**Pull request by sigma-andex**
### Description of the change

Update the README.md to contain examples for v0.15.0. Solves #863 .

### Checklist:

- [x] Added the change to the "Unreleased" section of the changelog
- [x] Added some example of the new feature to the `README`
- [ ] Added a test for the contribution (if applicable)

**P.S.**: the above checks are not compulsory to get a change merged, so you may skip them. However, taking care of them will result in less work for the maintainers and will be much appreciated 😊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -398,10 +398,8 @@ Tests succeeded.
 As with the `build` and `test` commands, you can add custom source paths
 to load, and pass options to the underlying `purs repl` via `--purs-args`.
 
-E.g. the following opens a repl on `localhost:3200`:
-
 ```bash
-$ spago repl --purs-args "--port 3200"
+$ spago repl
```
Comment (thomashoneyman) [inline README.md:402]: If there are no longer arguments that can be provided to the repl, should this section be reworded or removed altogether?

File: README.md [hunk 2]
```diff
@@ -769,6 +767,27 @@ there are basically two ways to do that:
 
 This will produce a single, executable, dead-code-eliminated file:
 
+**>= v0.15.0**
+
+Since v0.15.0 spago uses `esbuild` as the underlying default bundler. See the [`esbuild` getting started](https://esbuild.github.io/getting-started/#install-esbuild) for installation instructions.
+```bash
```
Comment (thomashoneyman) [inline README.md:773]:
```suggestion
```console
```

Comment (thomashoneyman) [reply README.md:773]: I suppose all of the code blocks in this README use `bash`, but they ought to use `console` if they're indicating commands to run and printed results. Feel free to ignore this and we can update those later if you wish.

File: README.md (hunk 1 above)
Comment (JordanMartinez) [reply README.md:402]: I'd remove it. Even if it technically would still work on `purs-0.14`, it's not a well-known feature.

Comment (sigma-andex) [reply README.md:401]: I rephrased it since the `repl` command is still valid.

File: README.md (hunk 2 above)
Comment (sigma-andex) [reply README.md]: Done.

**Comment by sigma-andex** [comment]
@f-f can we merge this as well now?

**Comment by f-f** [comment]
Right, it looks like this might need a rebase?

**Comment by JordanMartinez** [comment]
CI builds now.

**Pull request merged**